CHANGELOG.md

```diff
@@ -1347,6 +1351,11 @@ New modules
   Algebra.Module.Core
   ```
 
+* Definitions for module-like algebraic structures with left- and right- multiplication over the same scalars:
+  ```
+  Algebra.Module.Definitions.Bi.Simultaneous
```

Comment:
`Algebra.Module.Definitions.Bi.Commutative`?

Reply:
Should this now be `...Bi.Coincident`?

Reply:
I'm not convinced that this module's name ought to be changed

Reply:
Agreed. The one definition in it happens to be named `Coincident` but that shouldn't affect the name of the module.

Reply:
Fair enough, but since even the module name was up for grabs, I wondered about what we might think was 'best'. But very happy to see this merged, and look forward to downstream PRs where we have used these ideas, and thought some more about how best they should be deployed. Thanks @Taneb for the hard work on this!